### PR TITLE
Update file outdated korean docs in dev-1.16-ko.6.

### PR DESCRIPTION
**Pull request by ysyukr**
### **17 out of 24 files to be modified**
- [x] content/ko/docs/concepts/_index.md
- [x] content/ko/docs/concepts/architecture/cloud-controller.md
- [x] content/ko/docs/concepts/architecture/nodes.md
- [x] content/ko/docs/concepts/overview/components.md
- [x] content/ko/docs/concepts/overview/what-is-kubernetes.md
- [x] content/ko/docs/concepts/overview/working-with-objects/labels.md
- [x] content/ko/docs/concepts/workloads/controllers/daemonset.md
- [x] content/ko/docs/concepts/workloads/controllers/deployment.md
- [x] content/ko/docs/setup/_index.md
- [x] content/ko/docs/setup/learning-environment/minikube.md
- [x] content/ko/docs/setup/production-environment/tools/kops.md
> #17709 에서 일부 수정됨
- [x] content/ko/docs/setup/production-environment/windows/user-guide-windows-nodes.md
- [x] content/ko/docs/tasks/access-application-cluster/access-cluster.md
> #17709 에서 일부 수정됨
- [x] content/ko/docs/tasks/access-application-cluster/web-ui-dashboard.md
- [x] content/ko/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md
- [x] content/ko/docs/tasks/tools/install-minikube.md
- [x] content/ko/docs/tutorials/clusters/apparmor.md

#### #17626 에서 수정됨
- [ ] content/ko/case-studies/openAI/index.html
- [ ] content/ko/case-studies/squarespace/index.html
- [ ] content/ko/case-studies/ygrene/index.html
> #17709 에서 일부 수정됨
- [ ] content/ko/docs/contribute/participating.md
- [ ] content/ko/docs/reference/glossary/istio.md

#### #17544 에서 수정됨
- [ ] content/ko/docs/concepts/containers/runtime-class.md
#### #17570 에서 수정됨
- [ ] content/ko/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk.md

from #17755
/language ko

### CODE REVIEW

File: content/ko/docs/setup/_index.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ card:
 | [Docker Enterprise](https://www.docker.com/products/docker-enterprise) |       |&#x2714;  | &#x2714;     |  |     | &#x2714;
 | [Fedora (멀티 노드)](https://kubernetes.io/docs/getting-started-guides/fedora/flannel_multi_node_cluster/)     |      |  |      | |    &#x2714;       | &#x2714;
 | [Fedora (단일 노드)](https://kubernetes.io/docs/getting-started-guides/fedora/fedora_manual_config/)     |      |  |      | |           | &#x2714;
-| [Gardener](https://gardener.cloud/) | &#x2714; | &#x2714; | &#x2714; (via OpenStack) | &#x2714; | |
+| [Gardener](https://gardener.cloud/) | &#x2714; | &#x2714; | &#x2714; | &#x2714; | &#x2714; | [Custom Extensions](https://github.com/gardener/gardener/blob/master/docs/extensions/overview.md) |
```
Comment (sftim) [inline content/ko/docs/setup/_index.md]: Translate “Custom Extensions”?

Comment (ysyukr) [reply content/ko/docs/setup/_index.md]: @sftim Thank you for the missing part of the translation.

**Comment by gochist** [comment]
@ysyukr 감사합니다!
/lgtm
/approve

**Pull request merged**